internal/service/cluster/resource_cluster.go

```diff
@@ -984,7 +1009,15 @@ func resourceMongoDBAtlasClusterUpdate(ctx context.Context, d *schema.ResourceDa
 		}
 	}
 
-	return resourceMongoDBAtlasClusterRead(ctx, d, meta)
+	if d.HasChange("redact_client_log_data") {
+		if v, ok := d.GetOk("redact_client_log_data"); ok {
+			if ret := newAtlasUpdate(ctx, d.Timeout(schema.TimeoutUpdate), conn, connV2, projectID, clusterName, v.(bool)); ret != nil {
```

Comment:
`ret`? Why not `err`?

Reply:
oops, thanks for the catch. changed in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2601/commits/9aef32a65ab7c940fe8a6b5bc57c82e130f264c3